app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -268,8 +268,8 @@ def serials
     end
 
     def get_parent_host(headers)
-      hostnames = headers["HTTP_X_FORWARDED_SERVER"]
-      host = hostnames.split(",")[0] if hostnames
+      hostnames = headers["HTTP_X_FORWARDED_HOST"]
+      host = hostnames.split(/[\,,:]/)[0] if hostnames
```

Comment:
What's the difference between your regex and the `split` approach?

Reply:
Ah, ok! Can you include such an example in your unit tests? That'll make sure we don't break this later